spec/unit/lib/wifi_user/repository/user_spec.rb

```diff
@@ -46,6 +46,18 @@
     end
   end
 
+  context 'avoiding usernames including aeiou' do
```

Comment:
Might be good to say why we are doing this

context 'avoiding potentially offensive random words' do
    it 'generates usernames without vowels' do